.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Python Tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements-dev.txt
+        pip install pytest azure-functions coverage
```

Comment:
pytest and azure-functions are in requirements-dev.txt aren't they? Is it redundant to have this again?

Reply:
Good to try removing this line and adding coverage to requirements-dev.txt ?